Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -312,12 +312,24 @@ MainWidget::MainWidget()
         }
     });
 
+    auto file_manager_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/app-file-manager.png"sv).release_value_but_fixme_should_propagate_errors();
+    m_open_folder_action = GUI::Action::create("Open Containing Folder", { Mod_Shift | Mod_Alt, Key_R }, file_manager_icon, [&](auto&) {
```

Comment:
It might make more sense to change the shortcut to Ctrl+Shift+O (to complement Ctrl+O as an "open" action).

Reply:
I would be happy to change it, I expected some feedback on this. I used the same shortcut as VSCode in my test :)